src/engine/instrument.cpp

```diff
@@ -363,6 +363,153 @@ void DivInstrument::writeFeatureFM(SafeWriter* w, bool fui) {
   FEATURE_END;
 }
 
+void MemPatch::clear() {
+  data = NULL;
```

Comment:
Possible memory leak by not deleting data?

Reply:
no leak the way it's used but i agree it's ambiguous, have removed the function